etc/ghostwriter.profile

```diff
@@ -48,3 +48,8 @@ private-dev
 # passwd,login.defs,firejail are a temporary workaround for #2877 and can be removed once it is fixed
 private-etc alternatives,ca-certificates,crypto-policies,dbus-1,dconf,firejail,fonts,gconf,groups,gtk-2.0,gtk-3.0,host.conf,hostname,hosts,ld.so.cache,ld.so.conf,ld.so.conf.d,ld.so.preload,locale,locale.alias,locale.conf,localtime,login.defs,machine-id,mime.types,nsswitch.conf,pango,passwd,pki,protocols,resolv.conf,rpc,services,ssl,texlive,Trolltech.conf,X11,xdg
 private-tmp
+
+dbus-user filter
+dbus-user.own io.github.wereturtle.ghostwriter
+dbus-user.talk com.canonical.AppMenu.Registrar
```

Comment:
I think this would be needed for any Gnome application for top menu support, as well as for Qt applications with top menu support on X11. Top menu support on Wayland seems much more complicated, however (I was musing on that earlier: https://www.reddit.com/r/swaywm/comments/fn1uf9/global_app_menus/ , but the protocols are a mess of DBus and custom Wayland extensions).

Reply:
Maybe we should just drop `com.canonical.AppMenu.Registrar` from profiles and add a FAQ point to use globals.local if wanted.